Classes/common/query/CDTQIndex.m

```diff
@@ -0,0 +1,174 @@
+//
+//  CDTQIndex.m
+//
+//  Created by Al Finkelstein on 2015-04-20
+//  Copyright (c) 2015 Cloudant. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+//  except in compliance with the License. You may obtain a copy of the License at
+//    http://www.apache.org/licenses/LICENSE-2.0
+//  Unless required by applicable law or agreed to in writing, software distributed under the
+//  License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+//  either express or implied. See the License for the specific language governing permissions
+//  and limitations under the License.
+
+#import "CDTQIndex.h"
+
+#import "CDTQLogging.h"
+
+NSString *const kCDTQJsonType = @"json";
+NSString *const kCDTQTextType = @"text";
```

Comment:
These are externs.  I don't believe that I can declare them as both extern and static.